Makefile

```diff
@@ -57,14 +56,19 @@ package-goo:
 	goopack -output_dir ./dist <(envsubst < ./.build/googet/google-cloudops-opentelemetry-collector.goospec)
 	chmod -R 777 ./dist/
 
+# exporters
+.PHONY: build-exporters
+build-exporters:
+	bash ./.build/tar/build_exporters.sh
+
 # tarball
 
 .PHONY: build-tarball
-build-tarball: build package-tarball
+build-tarball: build build-exporters package-tarball
 
 .PHONY: package-tarball
 package-tarball:
-	./.build/tar/generate_tar.sh
+	bash ./.build/tar/generate_tar.sh
```

Comment:
Why is that change needed?

Reply:
This was something I wanted to change also. When we ran this, especially in the docker one, I always get the permission denied error message. Even if I do a chmod -x command, I still get a permission denied, but if I add bash here, it runs successfully for me. Is there a convention for such things?

Reply:
Not sure, but that seems fine for now